### PR TITLE
Better Logging for start/stop

### LINKED ISSUE #284: Logging Improvements
For all message handlers, we should have 

"Handler xx Dispatching Message Id #"

and then for plugins/etc it should be what it is but include the message ID

Also we should log as JSON, not a sentence, for easier queries

### PR DESCRIPTION
**Pull request by tanndlin**
Refs #284

### CODE REVIEW

File: src/app/handlers/command.handler.ts [hunk 1]
```diff
@@ -34,12 +34,16 @@ export class CommandHandler implements IHandler {
 
       try {
         this.container.loggerService.info(
-          `Running ${plugin.name} with args: {${command.args.join(', ')}}'`
+          JSON.stringify({ event: 'starting', plugin: plugin.name, args: command.args })
         );
         await plugin.execute(message, command.args);
-        this.container.loggerService.info(`The ${plugin.name} has been fulfilled`);
+        this.container.loggerService.info(
+          JSON.stringify({ event: 'fulfill', plugin: plugin.name, args: command.args })
```
Comment (c650) [inline src/app/handlers/command.handler.ts:41]: fulfillCommand

**Review by c650 — COMMENTED** [review]
I wouldnt say that this fixes the issue because there are lots of logs to convert to structured iirc

**Comment by c650** [comment]
but ill happily accept this as a `Refs`

**Review by c650 — COMMENTED** [review]
maybe we should introduce an Event type ? or PluginEvent?

**Comment by tanndlin** [comment]
> maybe we should introduce an Event type ? or PluginEvent?

can you explain a little

**Comment by c650** [comment]
> > maybe we should introduce an Event type ? or PluginEvent?
> 
> can you explain a little

Yeah. Maybe we should have a type so that typescript enforces certain fields that we want in all of our structured logs.

**Review by c650 — APPROVED** [review]

**Pull request merged**